### PR TITLE
feat: Hide tabs if only one is active.

### PR DESCRIPTION
**Pull request by stephenh**
This matches a change @bdow made to the old BP tabs, requested by Lauren. It should also help some of our create flows that "open in drawer, only enable the details tab" hopefully for-free match the mocks, where the "in drawer" create flows leave out the tabs.

Granted, sometimes the different between create-flow vs. update-flow is more than just "don't show the other tabs", but hopefully this is a good default.

### CODE REVIEW

File: src/components/Tabs.test.tsx [hunk 1]
```diff
@@ -74,6 +74,21 @@ describe("TabsWithContent", () => {
...
+      { name: "Tab 1", value: "tab1", render: () => <TabContent title="Tab 1 Content" /> },
+      { name: "Tab 2", value: "tab2", disabled: true, render: () => <TabContent title="Tab 2 Content" /> },
+      { name: "Tab 3", value: "tab3", disabled: true, render: () => <TabContent title="Tab 3 Content" /> },
+      { name: "Tab 4", value: "tab4", disabled: true, render: () => <TabContent title="Tab 4 Content" /> },
+    ];
+    const r = await render(<TabsWithContent tabs={testTabs} onChange={() => {}} selected="tab1" />);
+    // Then the tabs are not even in the dom
+    expect(() => r.tabs_tab1()).toThrow("Unable to find an element");
+    // But the 1st tab's content is
+    expect(r.tab_panel()).toHaveTextContent("Tab 1 Content");
```
[the remark this answers is not in the document]
Comment (bdow) [reply src/components/Tabs.test.tsx:90]: I guess your `toThrow` above this handles the case, but probably still good to be explicit about wanting the text content to be this and _only_ this text.

[the remark this answers is not in the document]
Comment (stephenh) [reply src/components/Tabs.test.tsx:90]: Yep, updating!

**Review by bdow — APPROVED** [review]

**Comment by bdow** [comment]
Thanks for making that change. I swear I was going to get around to it!

**Comment by stephenh** [comment]
> Thanks for making that change. I swear I was going to get around to it!

Np, I've got a ticket that could use it, and doing small & quick PRs is fun. :-)

**Pull request merged**